xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/actions/BaseAction.vue

```diff
@@ -0,0 +1,62 @@
+<!--
+  See the NOTICE file distributed with this work for additional
+  information regarding copyright ownership.
+  This is free software; you can redistribute it and/or modify it
+  under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation; either version 2.1 of
+  the License, or (at your option) any later version.
+  This software is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public
+  License along with this software; if not, write to the Free
+  Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<template>
+  <div
+    class="livedata-base-action btn"
+    @click="handler"
+  >
+    <XWikiIcon :icon-descriptor="icon" />
+    {{ $t("livedata.displayer.actions." + id) }}
```

Comment:
We need to discuss on the forum if we want to have labels in addition to the icons. That do not to have a title attribute on the image for the accessibility.